testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BruteForceTest.java

```diff
@@ -539,6 +547,26 @@ public void testEmail() throws Exception {
         expectTemporarilyDisabled("user2", userId);
         clearAllUserFailures();
     }
+    
+    @Test
+    public void testUserResourceUserDisabled() throws Exception {
+        String userId = adminClient.realm("test").users().search("test-user@localhost", null, null, null, 0, 1).get(0).getId();
+
+        loginInvalidPassword();
+        loginInvalidPassword();
+        expectTemporarilyDisabled();
+
+        String urlA = suiteContext.getAuthServerInfo().getContextRoot().toString() + "/auth/admin/realms/test/users/" + userId;
+        SimpleHttp a = SimpleHttpDefault.doGet(urlA, httpClient).auth(getAdminToken());
+
+        String urlB = suiteContext.getAuthServerInfo().getContextRoot().toString() + "/auth/admin/realms/test/users?username=test-user@localhost";
+        SimpleHttp b = SimpleHttpDefault.doGet(urlB, httpClient).auth(getAdminToken());
+
+        UserRepresentation user = a.asJson(UserRepresentation.class);
+        assertFalse(user.isEnabled());
+        user = b.asJson(new TypeReference<List<UserRepresentation>>() {}).get(0);
+        assertFalse(user.isEnabled());
```

Comment:
@douglaspalmer You don't need a `httpClient` here, you can use directly the `adminClient`:

```java
        assertFalse(testRealm().users().get(userId).toRepresentation().isEnabled());
        assertFalse(testRealm().users().search("test-user@localhost", true).get(0).isEnabled());
```